src/govuk/components/pagination/pagination.yaml

```diff
@@ -0,0 +1,169 @@
+params:
+- name: previous
+  type: object
+  required: false
+  description: A link to the previous page, if there is one.
+  params:
+  - name: text
+    type: string
+    required: false
+    description: The previous page link text. Defaults to 'Previous page', where 'page' is visually hidden.
```

Comment:
Does "previous page link text" mean the link text ON the previous page, or the link text TO the previous page?

If the former, then could change this to:

> The previous page's link text.

If the latter, then could change this to:

> The link text to the previous page.